src/app/app.component.css

```diff
@@ -0,0 +1,35 @@
+.image{
+	min-height:100vh;
```

Comment:
this is not right way to set parameter of image, 100vh bigger the 100% on mobile devices